fission-web-server/library/Fission/Web/Server/PowerDNS.hs

```diff
@@ -0,0 +1,11 @@
+module Fission.Web.Server.PowerDNS
+  ( module Fission.Web.Server.PowerDNS.Class
+  , getValuesFromRecords
+  ) where
+
+import           Fission.Prelude
+import           Fission.Web.Server.PowerDNS.Class
+import           PowerDNS.Client                   as PDNS
+
+getValuesFromRecords :: [SearchResult] -> Maybe (NonEmpty Text)
+getValuesFromRecords rrs = nonEmpty (PDNS.sr_content <$> rrs)
```

Comment:
@walkah let me know if you explicitly do not want the rest of the values that are in the `[SearchResult]`. This version keeps them available rather than tossing them. Being a lazy language, we'll only get the ones we consume.